cg/cli/status.py

```diff
@@ -377,11 +377,11 @@ def samples(context: CGConfig, skip: int):
 @status.command()
 @click.option("-s", "--skip", default=0, help="skip initial records")
 @click.pass_obj
-def families(context: CGConfig, skip: int):
+def get_cases(context: CGConfig, skip: int):
     """View status of families."""
     click.echo("red: prio > 1, blue: prio = 1, green: completed, yellow: action")
     status_db: Store = context.status_db
-    records: List[Family] = status_db.families().offset(skip).limit(30)
+    records: List[Family] = status_db.get_filtered_cases().offset(skip).limit(30)
```

Comment:
get filtered cases might be a bit too vague? What are they filtered for?

Reply:
I agree, maybe the function can be split into several smaller ones.